### PR TITLE
Bug fix in LockManager loop

### PR DESCRIPTION
**Pull request by hbdeshmukh**
- Added a false condition for acquiring lock
- Added clarifying comment.

### CODE REVIEW

File: transaction/LockManager.cpp [hunk 1]
```diff
@@ -81,15 +81,17 @@ void LockManager::run() {
           CHECK(releaseAllLocks(request.getTransactionId()))
               << "Unexpected condition occured.";
 
-        } else if (acquireLock(request.getTransactionId(),
```
Comment (hakanmemisoglu) [inline transaction/LockManager.cpp]: Good catch!

**Comment by hbdeshmukh** [comment]
Fixed the style issues and did minor refactoring.

File: transaction/LockManager.cpp [hunk 2]
```diff
@@ -79,22 +79,23 @@ void LockManager::run() {
         const LockRequest request = incoming_requests_.popOne();
         if (request.getRequestType() == RequestType::kReleaseLocks) {
           CHECK(releaseAllLocks(request.getTransactionId()))
-              << "Unexpected condition occured.";
-
+          << "Unexpected condition occured.";
```
Comment (zuyu) [inline transaction/LockManager.cpp]: I think the original style is correct.

File: transaction/LockManager.cpp [hunk 3]
```diff
@@ -79,22 +79,23 @@ void LockManager::run() {
...
         } else if (acquireLock(request.getTransactionId(),
-                               request.getResourceId(),
-                               request.getAccessMode())) {
+                                request.getResourceId(),
+                                request.getAccessMode())) {
```
Comment (zuyu) [inline transaction/LockManager.cpp]: Please align with the first argument.

**Comment by hbdeshmukh** [comment]
Fixed the style issues.

**Comment by zuyu** [comment]
LGTM!

**Pull request merged**